superset-frontend/src/logger/LogUtils.ts

```diff
@@ -56,6 +65,14 @@ export const LOG_EVENT_TYPE_USER = new Set([
   LOG_ACTIONS_MOUNT_EXPLORER,
 ]);
 
+export const LOG_EVENT_DATASET_TYPE_DATASET_CREATION = new Set([
```

Comment:
what is this set used for?

Reply:
Yeah I was trying to figure that out as well, because you end up just importing them straight into the component either way. It was the pattern used by the other logging actions above so I copied it, but those sets are also not invoked anywhere. Happy to delete.

Reply:
I removed the set in [`this commit`](https://github.com/apache/superset/pull/21241/commits/8e2b2c113f4e7080a76e1dff106ffccf43686104).